src/components/map/map.e2e.ts

```diff
@@ -53,7 +52,8 @@ describe("gx-map", () => {
     element = await page.find("gx-map");
     expect(await element.getProperty("mapType")).toEqual("standard");
   });
-  +it("should properly intereact with gx-map-marker components", async () => {
+
+  it("should properly intereact with gx-map-marker components", async () => {
```

Comment:
```suggestion
  it("should properly interact with gx-map-marker components", async () => {
```

Reply:
hehehe "inteReact" haha. My bad.
Fixing it inmediatly